### ISSUE TITLE
Make it clear if terms are selected

### ISSUE THREAD

**Issue by adityaekawade** [iobio/clin.iobio#345]
If you refine terms, we see a list of terms that can be expanded. We are told how many options there are under each heading, but not if any are actually selected. For example coarctation of the aorta has "1 more option". It doesn't - it has precisely one option. If there is only one option, should we have an expandable field? Regardless, if I expand, I see the term is already selected. Would be good to know that without having to go through and click to expand.

Genepanel issue: https://github.com/iobio/genepanel.iobio/issues/369

**Comment by adityaekawade** [comment]
<img width="1003" alt="Screen Shot 2021-02-18 at 1 42 31 PM" src="https://user-images.githubusercontent.com/16284713/108425422-47c2c000-71ef-11eb-944c-6b0a8a02b4fc.png">

Changed the wordings to show "1 option" instead of "1 more option".  It also shows the number of terms selected under each heading, so you now know the terms selected without having to go through and click to expand.

Regarding your comment about having an expandable field when there is only one option. I remember we once had a checkbox next to the heading and that was very confusing, so we had removed that and put everything under expandable panels. 
Test: https://dev.clin.iobio.io/
